src/debugging/inspector/Inspector/NativeScript Inspector/Communication.h

```diff
@@ -0,0 +1,43 @@
+#ifndef Communication_h
+#define Communication_h
+
+#import <Foundation/Foundation.h>
+#import <netinet/in.h>
+#import <sys/types.h>
+#import <sys/socket.h>
+#import <errno.h>
+#import <stdlib.h>
+#import <string.h>
+#import <notify.h>
+
+struct communication_channel {
+    BOOL connected;
+    dispatch_fd_t socket;
+    __unsafe_unretained dispatch_io_t io_channel;
```

Comment:
I think we should use a class instead of a struct so that ARC can manage the class instance and the `dispatch_io_t` object within.